<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgPathEdit</class>
 <widget class="QDialog" name="CQDlgPathEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Path edition</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Path properties</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="qqClosed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Path is closed</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqFlat">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Path is flat</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqKeepXup">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Keep X up</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqAutomaticOrientation">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Automatic orientation</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>361</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Last selected control point properties</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="qqFactor1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Bezier interpolation factor 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Bezier interpolation factor 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqFactor2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqPointCount">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Bezier point count</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqVirtualDistance">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Virtual distance [m]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqAuxFlags">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>56</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Auxiliary flags</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Auxiliary channels 1-2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqAuxChannel1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqAuxChannel2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqAuxChannel3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqAuxChannel4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Auxiliary channels 3-4</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Control point selection</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QToolButton" name="qqClearSelection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Clear selection</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqInvertSelection">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Invert selection</string>
    </property>
   </widget>
   <widget class="QLabel" name="qqInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>341</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Operations on selected control points</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QToolButton" name="qqMakeDummies">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Make dummies</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="qqApply">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>410</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>237</red>
         <green>237</green>
         <blue>237</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>110</red>
         <green>110</green>
         <blue>110</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>147</red>
         <green>147</green>
         <blue>147</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>237</red>
         <green>237</green>
         <blue>237</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>237</red>
         <green>237</green>
         <blue>237</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>110</red>
         <green>110</green>
         <blue>110</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>147</red>
         <green>147</green>
         <blue>147</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>237</red>
         <green>237</green>
         <blue>237</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>110</red>
         <green>110</green>
         <blue>110</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Light">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Midlight">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>237</red>
         <green>237</green>
         <blue>237</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Dark">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>110</red>
         <green>110</green>
         <blue>110</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Mid">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>147</red>
         <green>147</green>
         <blue>147</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>110</red>
         <green>110</green>
         <blue>110</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="BrightText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>110</red>
         <green>110</green>
         <blue>110</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Shadow">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="AlternateBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>220</red>
         <green>220</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipBase">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>220</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Apply to selection</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqClosed</tabstop>
  <tabstop>qqFlat</tabstop>
  <tabstop>qqAutomaticOrientation</tabstop>
  <tabstop>qqKeepXup</tabstop>
  <tabstop>qqClearSelection</tabstop>
  <tabstop>qqInvertSelection</tabstop>
  <tabstop>qqMakeDummies</tabstop>
  <tabstop>qqFactor1</tabstop>
  <tabstop>qqFactor2</tabstop>
  <tabstop>qqPointCount</tabstop>
  <tabstop>qqVirtualDistance</tabstop>
  <tabstop>qqAuxFlags</tabstop>
  <tabstop>qqAuxChannel1</tabstop>
  <tabstop>qqAuxChannel2</tabstop>
  <tabstop>qqAuxChannel3</tabstop>
  <tabstop>qqAuxChannel4</tabstop>
  <tabstop>qqApply</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
